docs/notebooks/visualizing_with_tensorboard.pct.py

```diff
@@ -62,16 +62,24 @@
 bo = trieste.bayesian_optimizer.BayesianOptimizer(observer, search_space)
 result, history = bo.optimize(num_steps, initial_data, model).astuple()
 
-
 # %% [markdown]
 # The data remains available when the optimization is over.
 #
 # ![TensorBoard graphs](figures/tensorboard_bo.png)
 
+# %% [markdown]
+# ## Deciding what to log
+#
+# When monitoring is enabled, Trieste decides what information is interesting enough to log. This includes objective and acquisition function values, query points, and some model parameters. Some data summaries, such as query point pairplots, are not generated by default as they can affect running time. You can control which summaries to generate and display by overriding the default summary filter. By default this shows any summary apart from those whose name starts with an underscore character.
+
+# %%
+trieste.logging.set_summary_filter(lambda name: True)  # enabe all summaries
+
+
 # %% [markdown]
 # ## Logging additional model parameters
```

Comment:
perhaps make one if these experiments with a batch size larger than 1, to illustrate that functionality
we should also mention that some of the summaries require Pandas and seabourn libraries